docs/admin_api/experimental_features.md

```diff
@@ -13,33 +13,42 @@ for a server admin: see [Admin API](../usage/administration/admin_api/).
 
 ## Enabling/Disabling Features
 
-This API allows a server administrator to enable experimental features for a given user, where the
-`user_id` is the user id of the user for whom to enable or disable the features. The request must 
-provide a body listing the features to enable/disable in the following format:
+This API allows a server administrator to enable experimental features for a given user. The request must 
+provide a body containing the user id and listing the features to enable/disable in the following format:
 ```
 {
-"features": {"msc3026": True, "msc2654": True}
+"user_id": user_id,
+"features": {"msc3026": true, "msc2654": true}
 }
 
 ```
-where True is  used to enable the feature, and False is used to disable the feature.
+where true is  used to enable the feature, and false is used to disable the feature.
 
 
 The API is:
 
 ```
-PUT /_synapse/admin/v1/experimental_features/<user_id>
+PUT /_synapse/admin/v1/experimental_features
```

Comment:
I think I would still keep the User ID in the URL because that's REST-y (you should imagine that `PUT` conceptually overwrites whatever is at the URL you're requesting, so as it stands you'd imagine this API would remove all experimental flags for everyone else as you use it. But that would be a little bit crazy :))